content/en/blog/_posts/2022-08-31-cgroupv2-ga.md

```diff
@@ -116,8 +116,10 @@ Scenarios in which you might need to update to cgroup v2 include the following:
   agents to versions that support cgroup v2.
 * If you run [cAdvisor](https://github.com/google/cadvisor) as a stand-alone
   DaemonSet for monitoring pods and containers, update it to v0.43.0 or later.
-* If you deploy Java applications with the JDK, prefer to use JDK 11.0.16 and
-  later or JDK 15 and later, which [fully support cgroup v2](https://bugs.openjdk.org/browse/JDK-8230305).
+* If you deploy Java applications, prefer to use versions which fully support cgroup v2:
+    * [HotSpot](https://bugs.openjdk.org/browse/JDK-8230305): jdk8u372, 11.0.16, 15 and later
```

Comment:
Should we say `Hotspot / Open JDK` ?

Reply:
I've always found the term OpenJDK ambiguous because, for example, IBM Semeru Runtimes also uses OpenJDK for its *JDK* implementation, but the issue here isn't about the *JDK classes* but rather about the JVM which does the cgroups detection (IBM's JVM is called J9). However, OpenJDK is also commonly used to refer to the whole package of JDK and HotSpot JVM, so I can see an argument for both sides. Do you happen to know anyone from Oracle that could comment?

Reply:
/cc @jmtd, I see you added the backport, do you have any thoughts here?

Reply:
It's a tricky one. I'd definitely choose to include "OpenJDK" in the wording, because we know that the support has landed there, and can't speak more broadly about downstream vendors' versions (Vendors are capable of removing or disabling features as well as adding them: although it's very unlikely for that to happen for a feature like this).  As for mentioning HotSpot: I'd argue it's less important: the majority of OpenJDK users are using HotSpot, but  many of them may be unfamiliar with the terminology; on the other hand, for users who *don't* use HotSpot, such as OpenJ9 users, I'd wager they are more likely to be aware that they can't take a statement about OpenJDK on face value for their situation.

Reply:
Okay, so correct me if I misunderstood, but it sounds like the preference is to just write "OpenJDK / HotSpot". I'll update the commit and squash soon (-ish; have a few higher priority things right now).

Reply:
Thanks @jmtd and @DanHeidinga for the feedback!